### PR TITLE
I4681 intellij inspections

### PR DESCRIPTION
**Pull request by romani**
#4681

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java [hunk 1]
```diff
@@ -575,7 +575,7 @@ public void testLoadConfigurationFromClassPath() {
             final Configuration[] children = config.getChildren();
             assertEquals(0, children[0].getChildren().length);
         }
-        catch (CheckstyleException ex) {
+        catch (CheckstyleException ignored) {
             fail("unexpected exception");
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java]: we should remove try-catch-fail.

File: src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java [hunk 2]
```diff
@@ -547,7 +547,7 @@ public void testLoadConfigurationDeprecated() {
             final Configuration[] children = config.getChildren();
             assertEquals(0, children[0].getChildren().length);
         }
-        catch (CheckstyleException | FileNotFoundException ex) {
+        catch (CheckstyleException | FileNotFoundException ignored) {
             fail("unexpected exception");
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java [hunk 3]
```diff
@@ -137,7 +137,7 @@ public void testTokenValueIncorrect2() throws Exception {
             assertEquals("given id " + id, expected.getMessage());
 
         }
-        catch (IllegalAccessException | NoSuchFieldException ex) {
+        catch (IllegalAccessException | NoSuchFieldException ignored) {
             fail("Exception is expected");
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java]: same.

Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java]: fixed

File: src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java (hunk 1 above)
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java]: fixed

File: src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java (hunk 2 above)
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java]: fixed

**Comment by romani** [comment]
manually squashed and rebase and merged

**Pull request closed without merging**